frontend/express/public/javascripts/countly/vue/components/sidebar.js

```diff
@@ -60,7 +115,8 @@
                     allApps: apps,
```

Comment:
after initialising allApps in global store.
allApps here should be a computed property which first gets all apps from global store.
```
allApps: function() {
    var apps = this.$store.getters["countlyCommon/getAllApps"];
    //Here you need to perform the sorting operation being done above and setting label and value etc
    //Remove this from data above and move here.
}
```

Reply:
@itsiprikshit  can we use mapGetters for this purpose ?

Reply:
the reason for making it a computed property is to make it reactive to vuex.
So when an app is added or deleted in vuex, allApps in component gets updated as it would react to the vuex changes.